### PR TITLE
modified the homepage to reflect the site redesign on other top level…

### PR DESCRIPTION
**Pull request by leseanbiz**
… pages.

### CODE REVIEW

File: client/app/components/HomePageTemplate/index.js [hunk 1]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-            Happy Customers
+    <div className="container">
+      <nav className="level">
+        <div className="level-item has-text-centered" style={{paddingTop: "3rem"}}>
+          <div>
+            <p className="heading">Happy Customers</p>
+            <div className="title">
+              <TimeCounter initialValue={385} interval={2500} increment={1} />
+            </div>
           </div>
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:75]: create loop out of these

File: client/app/components/HomePageTemplate/index.js [hunk 2]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-            <br />
-            Records Processed
+        </div>
+        <div className="level-item has-text-centered verticalCenter" style={{paddingTop: "3rem"}}>
+          <div>
+            <p className="heading">Records Processed</p>
+            <div className="title">
+              <TimeCounter initialValue={209} interval={1500} increment={1} />
+              <span> M</span>
+            </div>
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:83]: loop

File: client/app/components/HomePageTemplate/index.js [hunk 3]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-            Database Lookups
+        </div>
+        <div className="level-item has-text-centered verticalCenter" style={{paddingTop: "3rem"}}>
+          <div>
+            <p className="heading">Database Lookups</p>
+            <div className="title">
+              <TimeCounter initialValue={145} interval={500} increment={1} />
+              <span> M</span>
+            </div>
           </div>
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:93]: loop

File: client/app/components/HomePageTemplate/index.js [hunk 4]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-            <TimeCounter
-              initialValue={83874982374}
-              interval={100}
-              increment={111}
-            />
-            <br />
-            Data Points
+        </div>
+        <div className="level-item has-text-centered verticalCenter" style={{paddingTop: "3rem"}}>
+          <div>
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:96]: loop

File: client/app/components/HomePageTemplate/index.js [hunk 5]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-        <ThreeBlock
-          backgroundImage="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087377/idicia/Support-20.png"
-          icon="fa-phone"
-          title="Awesome support"
+
+      <ThreeBlock
+          backgroundImage="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087372/idicia/dataBackground-20.png"
+          icon="fa-database"
+          title="Fast & Accurate Data"
         >
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:119]: create loop out of these. If you copy paste code, it needs a loop.

File: client/app/components/HomePageTemplate/index.js [hunk 6]
```diff
@@ -159,49 +134,53 @@ const HomePageTemplate = ({ data }) => (
...
         </ThreeBlock>
+
         <ThreeBlock
-          backgroundImage="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087372/idicia/dataBackground-20.png"
-          icon="fa-database"
-          title="Fast & Accurate Data"
+          backgroundImage="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087377/idicia/Support-20.png"
+          icon="fa-phone"
+          title="Awesome support"
         >
```
Comment (Globolobo) [inline client/app/components/HomePageTemplate/index.js:142]: loop

File: client/app/components/HomePageTemplate/index.js [hunk 7]
```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
...
-              border: '2px solid #ddd',
-              fontSize: '1.2rem',
-              backgroundColor: '#eee',
-            }}
-          >
-            <TimeCounter initialValue={120938} interval={1000} increment={1} />
-            <br />
-            Happy Customers
+    <div className="container">
+      <nav className="level">
```
Comment (stern-shawn) [inline client/app/components/HomePageTemplate/index.js:68]: @leseanbiz FYI excellent use of the Bulma Level component, this is exactly the right use-case for that css and it looks sooo much better than the original design. Good job :+1:

Comment (leseanbiz) [reply client/app/components/HomePageTemplate/index.js:68]: Thanks! I did something right! :)

**Comment by leseanbiz** [comment]
The refactoring of this page to add loops and Seans note to make all blocks the same width have  both been moved to different stories.

**Pull request merged**